ci/scripts/run-fpga-tests.sh

```diff
@@ -50,41 +50,66 @@ test_args=""
 echo "tags: ${fpga_tags}"
 if [ "${fpga_tags}" == "cw310_sival_but_not_rom_ext_tests" ]
 then
-    # Only consider tests that are tagged `cw310_sival` but not tagged `cw310_sival_rom_ext`.
+    # Only consider tests that are tagged `cw310_sival` but that do not have corresponding
+    # test tagged `cw310_sival_rom_ext`.
+
     # The difficulty is that, technically, they are different tests since `opentitan_test` creates
     # one target for each execution environment. The following query relies on the existence
     # of the test suite created by `opentitan_test` that depends on all per-exec-env tests.
+    # This query only removes all tests that have sibling tagged `cw310_sival_rom_ext`. We
+    # then rely on test tag filters to only consider `cw310_sival`.
     ci/bazelisk.sh query \
     "
-        `# Find all test tagged cw310_sival that are dependencies of the test suite identified`
-        attr(
-            \"tags\",
-            \"cw310_sival([^_]|$)\",
-            deps(
-                `# Find all test suites depending on a test tagged cw310_sival` \
-                kind(
-                    \"test_suite\",
-                    rdeps(
-                        //...,
-                        `# Find all tests tagged cw310_sival`
-                        attr(\"tags\",\"cw310_sival([^_]|$)\", //...),
-                        1
-                    )
-                )
-                except
-                `# Remove all test suites depending on a test tagged cw310_sival_rom_ext`
-                rdeps(
-                    //...,
-                    `# Find all tests tagged cw310_sival_rom_ext`
-                    attr(\"tags\",\"cw310_sival_rom_ext\", //...),
-                    1
-                ),
+        `# Find all tests that are dependencies of the test suite identified`
+        deps(
+            `# Find all test suites`
+            kind(
+                \"test_suite\",
+                //...
+            )
+            except
+            `# Remove all test suites depending on a test tagged cw310_sival_rom_ext`
+            rdeps(
+                //...,
+                `# Find all tests tagged cw310_sival_rom_ext`
+                attr(\"tags\",\"cw310_sival_rom_ext\", //...),
```

Comment:
should this check for broken?

Reply:
Hum, that's a good point, we might not want to ignore a test if the corresponding `cw310_sival_rom_ext` is marked as broken? I will update this.